### ISSUE TITLE
[Bug]: Unable to authenticate

### ISSUE THREAD

**Issue by chenrupo** [Shopify/cli#4876]
### Please confirm that you have:

- [x] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [x] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Other

### Expected behavior

I should have clicked the link, logged in as shop owner, verified the code, and been granted access in the CLI

### Actual behavior

I clicked the login link, logged in as owner, verified the code, and got an error in the CLI

### Verbose output

```shell
error

You are not authorized to use the CLI to develop in the provided store.          
                                                                                   
You can't use Shopify CLI with development stores if you only have Partner       
staff member access. If you want to use Shopify CLI to work on a development     
store, then you should be the store owner or create a staff account on the       
store.                                                                           
                                                                                 
If you're the store owner, then you need to log in to the store directly using   
the store URL at least once before you log in using Shopify CLI.Logging in to    
the Shopify admin directly connects the development store with your Shopify      
login.
```

### Reproduction steps

1. Run command `shopify theme push --store storename`
2. Press any key to login
3. login as shop owner
4. Return to terminal

### Operating System

MacOs Sonoma 14.4.1

### Shopify CLI version (check your project's `package.json` if you're not sure)

3.70.0

### Shell

bash

### Node version (run `node -v` if you're not sure)

22.9.0

**Comment by arnoldski** [comment]
I'm having the same issue...

**Comment by mgmanzella** [comment]
👋 can you share the verbose output of the command after running it? we need more information to debug -- thanks in advance

**Comment by chenrupo** [comment]
@mgmanzella [verbose output.txt](https://github.com/user-attachments/files/17838044/verbose.output.txt)
thanks!

**Comment by chenrupo** [comment]
@lukeh-shopify Nope, I changed the shop name in the logs. The shop name I am using in the command is the same as the shopname I am logging into.

**Comment by Dexmente** [comment]
So no response yet, right @chenrupo ?

**Comment by mgmanzella** [comment]
👋 can you resend the logs in their original form with the command you're running to me in the partners slack? we need your real shop domain in order to debug so please don't modify them in any way, you can DM it to me at Morisa Manzella

**Comment by chenrupo** [comment]
@mgmanzella done, thanks!

**Comment by aswamy** [comment]
Hey @chenrupo, have you tried running the same command with your original domain for your shop? It can be found under "Settings > Domains". It should have "Redirects to <primary domain>" under it.

So your equivalent command would be: `shopify theme push --store <original_domain>`

**Comment by arnoldski** [comment]
Anybody an update on this issue?

**Comment by aswamy** [comment]
@arnoldski for now, the [temporary solution](https://github.com/Shopify/cli/issues/4876#issuecomment-2501508641) should fix the immediate problem. You can also set the environment variable (SHOPIFY_FLAG_STORE) before running `shopify theme push`.

**Comment by branlok** [comment]
Hello I'm also getting this issue suddenly mid-way development. I already authenticated once since yesterday. Today everything was going smoothly until I realize my changes weren't propagating, so I try to reauthenticate. 

ran `shopify theme dev --store  myshop` again

I followed the verification procedure. Then return to terminal I'm prompted with:

```
╭─ error ──────────────────────────────────────────────────────────────────────╮
│                                                                              │
│  You are not authorized to use the CLI to develop in the provided store.     │
│                                                                              │
│  You can't use Shopify CLI with development stores if you only have Partner  │
│   staff member access. If you want to use Shopify CLI to work on a           │
│  development store, then you should be the store owner or create a staff     │
│  account on the store.                                                       │
│                                                                              │
│  If you're the store owner, then you need to log in to the store directly    │
│  using the store URL at least once before you log in using Shopify           │
│  CLI.Logging in to the Shopify admin directly connects the development       │
│  store with your Shopify login.                                              │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯Ï
```

The workaround suggested above does not work for me.

**Comment by peepers-rick** [comment]
> Hello I'm also getting this issue suddenly mid-way development. I already authenticated once since yesterday. Today everything was going smoothly until I realize my changes weren't propagating, so I try to reauthenticate.
> 
> ran `shopify theme dev --store myshop` again
> 
> I followed the verification procedure. Then return to terminal I'm prompted with:
> 
> ```
> ╭─ error ──────────────────────────────────────────────────────────────────────╮
> │                                                                              │
> │  You are not authorized to use the CLI to develop in the provided store.     │
> │                                                                              │
> │  You can't use Shopify CLI with development stores if you only have Partner  │
> │   staff member access. If you want to use Shopify CLI to work on a           │
> │  development store, then you should be the store owner or create a staff     │
> │  account on the store.                                                       │
> │                                                                              │
> │  If you're the store owner, then you need to log in to the store directly    │
> │  using the store URL at least once before you log in using Shopify           │
> │  CLI.Logging in to the Shopify admin directly connects the development       │
> │  store with your Shopify login.                                              │
> │                                                                              │
> ╰──────────────────────────────────────────────────────────────────────────────╯Ï
> ```
> 
> Note: I'm using my own/shop owners account.
> 
> The workaround suggested above does not work for me.

I'm having the same problem as @branlok out of nowhere. I've been developing on this same store for years. It worked this morning, then out of nowhere this popped up. I've tried all the workarounds posted here and on the support forums with no luck.

**Comment by grocco** [comment]
There must be something wrong going on on Shopify's side; I am experiencing the same error right now out of nowhere mid development.

**Comment by aaronmarkle** [comment]
Also experiencing this error:

╭─ error ──────────────────────────────────────────────────────────────────────╮
│                                                                              │
│  You are not authorized to use the CLI to develop in the provided store.     │
│                                                                              │
│  You can't use Shopify CLI with development stores if you only have Partner  │
│   staff member access. If you want to use Shopify CLI to work on a           │
│  development store, then you should be the store owner or create a staff     │
│  account on the store.                                                       │
│                                                                              │
│  If you're the store owner, then you need to log in to the store directly    │
│  using the store URL at least once before you log in using Shopify           │
│  CLI.Logging in to the Shopify admin directly connects the development       │
│  store with your Shopify login.                                              │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯Ï

**Comment by valnovak** [comment]
Same issue!

**Comment by martin2810** [comment]
I also experiencing this error... It doesn't work. I have tried everything: https://store.myhsopify.com, without myshopify and the original domain name...

**Comment by joshkirk** [comment]
All of a sudden I can't log into the store I'm working on as a staff account. Same issue as all the other commenters, and the store URL hasn't changed.

**Comment by peepers-rick** [comment]
Lots of others commenting here: https://community.shopify.dev/t/getting-you-are-not-authorized-to-use-the-cli-to-develop-in-the-provided-store-error/2642/19

Someone who has access to the Shopify Partners slack indicated that they're aware of it and it's an infrastructure issue. Would be nice if https://shopifystatus.com could update us.

**Comment by martin2810** [comment]
It seems to work right now! :)

**Comment by patrick-mcdougle** [comment]
Anyone who has commented in the last 40 minutes should check again, seems to be working again.

**Comment by grundmanise** [comment]
It works for me now, although I'm not sure if it was related to the `shopify app info` command - as it started working at that time.

**Comment by maksym-mishchanchuk** [comment]
@aswamy 
You saved me, I couldn't fix this for 4 hours. Thank you
